src/Migrations/Version20190404131012.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20190404131012 extends AbstractMigration
+{
+    public function up(Schema $schema): void
+    {
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'postgresql', 'Migration can only be executed safely on \'postgresql\'.');
```

Comment:
I understand the need of making a clearance of Sequence but this is not the point of this PR, can you do it in another PR with a related issue explaining why those changes are needed please? thanks :)

Reply:
I can't because CI is not passing and then I can't merge.

Also if we merge with current errors, everyone else will have same errors in PR.

So I'm forced to clear migrations in same time I'm adding this CI check.

Reply:
works for me :)